Demos/FluentUIDemo_iOS/FluentUI.Demo/Demos/CommandBarDemoController.swift

```diff
@@ -339,17 +343,15 @@ class CommandBarDemoController: DemoController {
             ]
         ]
 
-        let itemGroups: [CommandBarItemGroup] = commandGroups.map { commandGroup in
-            commandGroup.map { command in
-                newItem(for: command)
-            }
+        let itemGroups: [CommandBarItemGroup] = commandGroups.map {
+            CommandBarItemGroup(items: $0.map { newItem(for: $0) })
         }
 
-        itemGroups[0][1].isEnabled = false
-        itemGroups[2][0].isEnabled = false
+        itemGroups[0].items[1].isEnabled = false
```

Comment:
Idea: were you interested in implementing the [Subscript](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/subscripts/) operator on `CommandBarItemGroup` so you can access values directly via square brackets?